### PR TITLE
docs: update PyPI Links

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did

Noticed that you can add additional links to a PyPI page (including funding) so I added some

https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#project-urls

### CODE REVIEW

**Review by NotPeopling2day — DISMISSED** [review]
🔥

File: setup.py [hunk 1]
```diff
@@ -80,7 +80,13 @@
     long_description_content_type="text/markdown",
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/ape",
+    url="https://apeworx.io",
+    project_urls={
+        "Documentation": "https://docs.apeworx.io/ape/",
+        "Funding": "https://gitcoin.co/grants/5958/ape-maintenance-fund",
+        "Source": "https://github.com/ApeWorX/ape",
+        "Tracker": "https://github.com/ApeWorX/ape/issues",
```
Comment (unparalleled-js) [inline setup.py:88]: If you add `Twitter` here, I think it will show up with the little bird! Wondering if there are any more?

**Review by unparalleled-js — DISMISSED** [review]

File: setup.py (hunk 1 above)
Comment (fubuloubu) [reply setup.py:88]: Twitter is a good one!

Comment (fubuloubu) [reply setup.py:88]:
```suggestion
        "Tracker": "https://github.com/ApeWorX/ape/issues",
        "Twitter": "https://twitter.com/ApeFramework",
```

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**